test/garbage.spec.js

```diff
@@ -6,15 +6,15 @@ const garbage = require('garbage')
 
 const cbor = require('../src')
 
-const CBOR_GARBAGE = process.env.NODE_CBOR_GARBAGE
-const NO_GARBAGE = process.env.NO_GARBAGE
-const REPEATS = parseInt(CBOR_GARBAGE || 1000)
+// const CBOR_GARBAGE = global.process.env.NODE_CBOR_GARBAGE
```

Comment:
Why not just removing those instead of commenting out?

Reply:
just wanted to confirm with the reviewer if this would be better removed

Reply:
this shouldn't be removed

Reply:
we need to be able to set these values when testing

Reply:
sure but these are tests and nothing is setting these env vars

how would you like this to be fixed ?

Reply:
Though it's neither document nor CI is using it. When is this test ever skipped (`NO_GARBAGE` is set)?

Reply:
the test does nothing probably some left over from history

Reply:
I used this, and manually set it when I working on a fast iteration cycle as this is a pertty slow test. Please preserve the functionality that is there, allowing configuration of this test through setting env variables